Marlin/example_configurations/delta/FLSUN/auto_calibrate/Configuration.h

```diff
@@ -450,8 +450,8 @@
   // See http://minow.blogspot.com/index.html#4918805519571907051
   #define DELTA_CALIBRATION_MENU
 
-  // set the radius for the calibration probe points - max 0.8 * DELTA_PRINTABLE_RADIUS if DELTA_AUTO_CALIBRATION enabled
-  #define DELTA_CALIBRATION_RADIUS (DELTA_PRINTABLE_RADIUS - 17) // mm
+  // set the radius for the calibration probe points - max DELTA_PRINTABLE_RADIUS*0.869 if DELTA_AUTO_CALIBRATION enabled
+  #define DELTA_CALIBRATION_RADIUS (DELTA_PRINTABLE_RADIUS*0.869) // mm
```

Comment:
Background: All options in Marlin may be set externally, via `Makefile`, shell environment, etc. Since we have no control over what values any configuration option will contain, we must wrap all options in parentheses when used in equations, comparisons, etc. Otherwise, a value like "`X_MIN_POS + 10`" may cause hard-to-find bugs.

The ability to override options from an external source is also the reason that we use `ENABLED` instead of `defined` for simple config options. It allows configuration options to be disabled by:
- being undefined (as we do in the configs), or
- by being set to `0` or `false` (as a `Makefile` would be required to do).